.github/runs-on.yml

```diff
@@ -1,11 +1 @@
-images:
-  aptos-ubuntu-x64:
-    platform: "linux"
-    arch: "x64"
-    ami: "ami-09f7d4bfb08b771be" # image for testing, with gcloud sdk and kubectl
-    owner: "058264343338" # aptos-ci AWS account id
-  aptos-ubuntu-x64-latest:
-    platform: "linux"
-    arch: "x64"
-    owner: "058264343338" # aptos-ci AWS account id
-    name: "aptos-runs-on-*"
+_extends: internal-ops
```

Comment:
nit: maybe include a link to this: https://runs-on.com/configuration/repo-config/. Always takes me a while to find the docs